src/components/GoalsCover.js

```diff
@@ -61,6 +61,11 @@ class GoalsCover extends React.Component {
     });
   }
 
+  componentDidMount() {
+    const state = this.populateGoals();
```

Comment:
I don't like this pattern. Why not have `getGoals()` and juste query that in `render()`?

Reply:
Render gets called on the backend and we don’t have the window size.

I don’t like that pattern either but my goal here was not to refactor, but just to show less goals on mobile.